src/js/shield_canvas_draw.js

```diff
@@ -420,9 +442,9 @@ export function hexagonVertical(
   let y0 = lineWidth;
   let y5 = CS - lineWidth;
 
-  let x1 = x0 + drawRadius;
+  // let x1 = x0 + drawRadius;
```

Comment:
Did you mean to leave these two comments?

Reply:
Commented lines of code removed; variable names decremented.